zboxcore/sdk/listworker.go

```diff
@@ -185,10 +185,10 @@ func (req *ListRequest) GetListFromBlobbers() *ListResult {
 					UpdatedAt: child.GetUpdatedAt(),
 				}
 				childResult.LookupHash = child.GetLookupHash()
+				childResult.Consensus.mu = &sync.RWMutex{}
 				childResult.consensus = 0
 				childResult.consensusThresh = req.consensusThresh
```

Comment:
It would be more convenient to init consensus with a function fo newConsensus(params...) instead of assign variables one by one everywhere.